src/main/kotlin/com/plus/taxiapp/api/member/MemberApi.kt

```diff
@@ -0,0 +1,65 @@
+package com.plus.taxiapp.api.member
+
+import com.plus.taxiapp.api.member.request.RegisterAccountRequest
+import com.plus.taxiapp.api.member.request.RegisterCardRequest
+import com.plus.taxiapp.api.member.response.RegisterAccountResponse
+import com.plus.taxiapp.api.member.response.RegisterCardResponse
+import com.plus.taxiapp.domain.PaymentService
+import com.plus.taxiapp.domain.command.PaymentCommand
+import com.plus.taxiapp.facade.MemberFacade
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/member/")
+class MemberApi(
+    private val memberFacade: MemberFacade,
+) {
+    @PostMapping("/register/account")
+    fun registerAccount(
+        @RequestBody request: RegisterAccountRequest,
+    ): RegisterAccountResponse {
+        val response = memberFacade.registerAccount(PaymentCommand.RegisterAccount(
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line